tools/build-tools/package.json

```diff
@@ -15,7 +15,8 @@
     "fluid-doc-stats": "bin/fluid-doc-stats",
     "fluid-layer-check": "bin/fluid-layer-check",
     "fluid-repo-policy-check": "bin/fluid-repo-policy-check",
-    "fluid-run-bundle-analyses": "bin/fluid-run-bundle-analyses"
+    "fluid-run-bundle-analyses": "bin/fluid-run-bundle-analyses",
+    "fluid-run-type-validator": "bin/typeValidator.ts"
```

Comment:
these don't get created automatically so you'll need to add one in the bin folder.  they're all thin wrappers so it should be pretty trivial to copy-edit one